Client/Frontend/AuthenticationManager/SensitiveViewController.swift

```diff
@@ -11,39 +11,39 @@ class SensitiveViewController: UIViewController {
     private var isAuthenticated = false
     private var willEnterForegroundNotificationObserver: NSObjectProtocol?
     private var didEnterBackgroundNotificationObserver: NSObjectProtocol?
-
+    
```

Comment:
I am pretty sure these will produce warnings. You have a bunch of them in the PR.